packages/volto/src/components/manage/Contents/ContentsUploadModal.jsx

```diff
@@ -388,7 +401,8 @@ export default compose(
   connect(
     (state) => ({
       request: state.content.subrequests?.[SUBREQUEST] || {},
+      uploadedFiles: state.content.uploadedFiles,
     }),
-    { createContent },
+    { createContent, updateUploadedFiles },
```

Comment:
@victorchrollo14 Where are we dispatching this action in this file?

Reply:
Thanks for pointing it out, I initially imported it thinking I'll be using in content upload modal, but then I didn't use it there and forgot to remove it.